app_layout_1.R

```diff
@@ -301,10 +313,14 @@ tab_selected_style = list(
 app <- Dash$new(external_stylesheets = dbcThemes$BOOTSTRAP)
 
 # layout made up of cards
-app$layout(htmlDiv(list(dbcContainer(htmlDiv(id="title_top",className="title on top", list(title()))), htmlDiv(dccTabs(id="tabs_in_app", value="tab-1"
-,list(dccTab(label="Game Dynamics Over Time", value='tab-1',children=list(dbcRow(list(dbcCol(first_card, width=3), dbcCol(second_card, width=9))), dbcRow(list(dbcCol(third_card, width=3),dbcCol(list(fourth_card,fifth_card), width=9))), dbcRow(dbcCol(ninth_card, width=3))), style=tab_style,
-selected_style=tab_selected_style),dccTab(label="Top Games", value="tab-2",children=list(dbcRow(list(dbcCol(sixth_card, width=3), dbcCol(list(seventh_card,eight_card), width=9)))),style=tab_style,
-selected_style=tab_selected_style)))))))
+app$layout(htmlDiv(list(dbcContainer(htmlDiv(id="title_top",className="title on top", list(title()))),
+htmlDiv(dccTabs(id="tabs_in_app", value="tab-1",list(dccTab(label="Game Dynamics Over Time", value='tab-1',
+children=list(dbcRow(list(dbcCol(first_card, width=3), dbcCol(second_card, width=9))),
+dbcRow(list(dbcCol(third_card, width=3),dbcCol(list(fourth_card,fifth_card), width=9)))), 
+style=tab_style,selected_style=tab_selected_style),
```

Comment:
recommend splitting this up a bit more/adding comments in, it's still pretty heavy